### PR TITLE
Fix pythreejs warning

### PR DESCRIPTION
**Pull request by dstansby**
I just ran across this trying out the new backend. Just a missing 's' 😄

### CODE REVIEW

File: pyvista/jupyter/pv_pythreejs.py [hunk 1]
```diff
@@ -442,13 +442,13 @@ def actor_to_mesh(actor, focal_point):
...
+            meshes.append(mesh)
         elif np.any(dataset.lines):
             mesh = to_edge_mesh(dataset, mapper, prop, use_edge_coloring=False,
                                 use_lines=True)
+            meshes.append(mesh)
         else:  # pragma: no cover
-            warning.warn(f'Empty or unsupported dataset attached to actor')
-
-        meshes.append(mesh)
+            warnings.warn(f'Empty or unsupported dataset attached to actor')
```
Comment (MatthewFlamm) [inline pyvista/jupyter/pv_pythreejs.py:451]:
```suggestion
            warnings.warn('Empty or unsupported dataset attached to actor')
```

**Review by MatthewFlamm — COMMENTED** [review]
I'm not really clear on how this backend works.  It is now possible that `meshes == []` when returning from this function.  I don't think this was possible before.  Will this cause other issues downstream?

**Comment by dstansby** [comment]
At least testing locally this fixes this block of code and outputs a sensible rendering for me.

**Comment by akaszynski** [comment]
Thanks for fixing this.  Will do a patch release for this.

**Review by akaszynski — APPROVED** [review]
Thanks for adding this.

**Comment by akaszynski** [comment]
> I'm not really clear on how this backend works. It is now possible that meshes == [] when returning from this function. I don't think this was possible before. Will this cause other issues downstream?

It's not an issue as we extend a list of meshes.  Extending with an empty list will have no effect.

**Pull request merged**